buildSrc/src/main/java/Dependencies.kt

```diff
@@ -176,6 +176,7 @@ object Deps {
     const val sentry = "io.sentry:sentry-android:${Versions.sentry}"
     const val leakcanary = "com.squareup.leakcanary:leakcanary-android-core:${Versions.leakcanary}"
 
+    const val androidx_compose_livedata = "androidx.compose.runtime:runtime-livedata:${Versions.androidx_compose}"
```

Comment:
This can probably be dropped in favor of using the simpler coroutines.flow.StateFlow api.